.travis.yml

```diff
@@ -1,22 +1,43 @@
-sudo: false
-language: scala
-addons:
-  hostname: localhost
-jdk:
-  - oraclejdk8
-scala:
-  - 2.11.12
-  - 2.12.8
-before_install:
-  - docker pull daunnc/openjdk-gdal:2.4.0
+# Required to clean up sbt caches written by container image as root
+sudo: required
```

Comment:
I'm not entirely comfortable with the approach of enforcing `$ sudo` on Travis. 

At the same time, I think this approach is cleaner than trying to get the container's UID/GID to match up with the CI environment.

Reply:
I spent hours on trying to speed up GeoTrellis builds; i definitely can say that it's not recommended by travis anymore to use `required`, they are moving towards deprecating this key word: https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

Also wondering what kind of permission errors it threw to you?

Reply:
That's what I was afraid of. 

The container drops files written as root, so I was getting a permission denied error when trying to delete them as non-root.

Reply:
@rbreslow what about our geotrellis build file? I haven't noticed any errors: https://github.com/locationtech/geotrellis/blob/master/.travis.yml